CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 **SDK**
 - **Feature:** Mini App can call media execution play/pause programmatically.
 - **Change:** Added the default implementation for external link handler. Using [custom tab](https://developers.google.com/web/android/custom-tabs).
+- **Removed:** The SDK setting `Host App Version` has been removed. There is no usage of it.
```

Comment:
Should we update this line as we have `hostAppVersionId` in `MiniAppSdkConfig`?

Reply:
I have removed this line. I guess it's the most proper way 😆.

Reply:
Okay, I think that's should be fine 👌